lib/flame/controller/actions.rb

```diff
@@ -39,15 +39,15 @@ def inherit_actions(actions = superclass.actions, exclude: [])
 			##     include with_actions Module2
 			##     ....
 			##   end
-			def with_actions(mod, exclude: [])
+			def with_actions(mod, exclude: [], only: nil)
```

Comment:
I think we should update the YARD above.

Reply:
updated

Reply:
You forgot about `@param`.